Over21/ViewController.swift

```diff
@@ -56,12 +72,19 @@ class ViewController: UIViewController {
     
     private func setupUIElements() {
         view.addSubview(ageIndicatorView)
+        view.addSubview(ageLimitSelectionView)
+        
         ageIndicatorView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
         ageIndicatorView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
         ageIndicatorView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         ageIndicatorView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
         
         
+        ageLimitSelectionView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        ageLimitSelectionView.topAnchor.constraint(equalTo: ageIndicatorView.scannerConnectionLabel.bottomAnchor, constant: 12).isActive = true
+        ageLimitSelectionView.widthAnchor.constraint(equalToConstant: 160.0).isActive = true
+        //ageLimitSelectionView.heightAnchor.constraint(equalToConstant: view.bottomAnchor).isActive = true
```

Comment:
should we remove this commented out code?

Reply:
Removed in latest commit.

Reply:
Did you push it because I don't see it removed?

Reply:
Yea, I pushed this change 4 minutes ago.